### ISSUE TITLE
ChangeOutputDirectory for an FdScript

### ISSUE THREAD

**Issue by xRadne** [strusoft/femdesign-api#397]
I wish there were a method to simply update the output directory for the files created from a FdScript.

Lets say I have a FdScript named `fdScript` and want write the files to the directory `outputDirectory` instead of the default (`Path.GetDirectoryName(strPath)`).

Currently I have to do the following:
```c#
// Hacky way to update the folder where the results etc will be saved.
fdScript.FdScriptHeader.LogFile = Path.Combine(outputDirectory, Path.GetFileNameWithoutExtension(strPath), Path.GetFileName(fdScript.FdScriptHeader.LogFile));
fdScript.CmdSave.FilePath = Path.Combine(outputDirectory, Path.GetFileName(fdScript.CmdSave.FilePath));
fdScript.FdScriptPath = Path.Combine(outputDirectory, Path.GetFileNameWithoutExtension(strPath), "scripts", Path.GetFileName(fdScript.FdScriptPath));
fdScript.CmdListGen.ForEach(c =>
{
    var resultsDir = Path.Combine(outputDirectory, Path.GetFileNameWithoutExtension(strPath), "results");
    if (!Directory.Exists(resultsDir))
        Directory.CreateDirectory(resultsDir);
    c.OutFile = Path.Combine(resultsDir, Path.GetFileName(c.OutFile));
});
```

I wish it was as simple as writing
```C#
fdScript.ChangeOutputDirectory(outputDirectory);
```

**Comment by xRadne** [comment]
The above code is written for a client which will recieve the full source code for further development themselves and they want the .str file to be in a read only directory. This means that no output files can be written to the default output structure in the same working directory.

**Comment by xRadne** [comment]
Already implemented by #546 

Output directory can be changed on the `FemDesignConnection` like this:
https://github.com/strusoft/femdesign-api/blob/a338ed7602c19cace46751ea551096686b5a02cc/FemDesign.Examples/C%23/Example%203%20-%20Analyse%20a%20beam/Program.cs#L125
